### ISSUE TITLE
Need to have BETTER PRIORS!

### ISSUE THREAD

**Issue by mortonjt** [mortonjt/q2-matchmaker#24]
So apparently the defaults that we had in place were garbage.

The crux here is that we are trying to fit something that smells like a logistic normal distribution to a count distribution (i.e. NB, PLN, Poisson, ...).  

If we have normal priors on the log-fold changes that's fine (and actually preferable).  The problem arises when we are trying to chose priors for our intercepts.  The intercepts are really the averaged log proportions of your control samples (see [here](https://mortonjt.github.io/probable-bug-bytes/probable-bug-bytes/regression-intercept/)).  If we want to model the intercepts correctly, we can't have them centered around zero -- its way off target.  This sort of prior misspecification will lead to overflow in the generated quantities that we've been seeing here : https://github.com/gibsramen/BIRDMAn/issues/45

Then, what's a good choice of prior?  You can answer this question by pulling out any random amplicon biom table and visualizing the stats.  Namely, we can look at all of the non-zero counts, compute their log proportions and visualize the distribution

```python
from biom import load_table
import seaborn as sns
from skbio.stats.composition import closure
import numpy as np

table = load_table('all.biom')
mat = np.array(table.matrix_data.todense())
mat.sum(axis=1)
mat.mean(axis=0)
pmat = closure(mat)
logmat = np.ma.log(pmat)
data = logmat.ravel()
data = data[~data.mask]
print(data.ravel().mean(), data.ravel().std())
sns.distplot(data)
```

![image](https://user-images.githubusercontent.com/4184797/119590149-72de2200-bd91-11eb-9fbc-6f19b880ee90.png)

So you can see, the normal prior is still not a bad choice, but note that it is centered at -5.5 with a standard deviation of 1.5.
Having this centered at zero *is a bad idea*.  From my tests, I'm finding that recentering the bias terms around -5.5 drastically improves model performance.

TL;DR centering regression coefficients around zero is fine, but centering all of the intercept / bias terms around zero is a horrible idea. @gibsramen
